Source/DafnyCore/AST/Tokens.cs

```diff
@@ -99,6 +99,7 @@ public static string RangeToFileString(this TokenRange range) {
 /// for quantified variables. See QuantifierVar.ExtractSingleRange()
 /// </summary>
 public class QuantifiedVariableDomainOrigin : OriginWrapper {
+  [SyntaxConstructor]
```

Comment:
Why did you mark this as `[SyntaxConstructor]` ? I don't think this type is part of the syntax of Dafny.

Reply:
Removed (context: https://github.com/dafny-lang/dafny/pull/6163#discussion_r2024061273)